app/views/early_career_payments/claims/eligible_itt_subject.html.erb

```diff
@@ -21,7 +22,7 @@
             <%= errors_tag current_claim.eligibility, :eligible_itt_subject %>
 
             <div class="govuk-radios">
-              <% eligible_itt_subjects.each do |option| %>
```

Comment:
L7 in this file, there is a reference to `_eligible_itt_subject_#{eligible_itt_subjects.first}" }) if`